src/data/mv_polynomial/basic.lean

```diff
@@ -626,6 +632,16 @@ lemma ne_zero_iff {p : mv_polynomial σ R} :
   p ≠ 0 ↔ ∃ d, coeff d p ≠ 0 :=
 by { rw [ne.def, eq_zero_iff], push_neg, }
 
+lemma eq_zero_iff_support_eq_empty (p: mv_polynomial σ R): p = 0 ↔ p.support = ∅ :=
```

Comment:
I think this would be better as
```suggestion
@{simp] lemma support_eq_empty_iff {p : mv_polynomial σ R} : p.support = ∅ ↔ p = 0 :=
```

Reply:
I just found that in `data.polynomial.basic` there is an `support_eq_empty`:
```lean
@[simp] lemma support_eq_empty : p.support = ∅ ↔ p = 0 :=
```
Could the similar one in `mv_polynomial` follow the name `support_eq_empty`?

Reply:
Good find. Yes, matching the names is almost always the right choice.